.gitignore

```diff
@@ -89,8 +89,11 @@ python/caffe/proto/
 cmake_build
 .cmake_build
 
-# tests
+# Test
 test/gnina/Testing/Temporary
+test/gnina/*.pdb
+test/gnina/*.xyz
+test/gnina/*.pymol
 
```

Comment:
testing tempfiles should be removed by the test that generated them at test completion; if you're generating other files in there you don't want to track, put this stuff in your local gitignore - it doesn't need to be in our project gitignore